.changeset/early-yaks-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+"@definitelytyped/dtslint-runner": patch
```

Comment:
I'm not sure if I should actually do this; don't we bump everything at once? So maybe I shouldn't do this at all and we're just using changesets to bump/publish?

Reply:
I have no idea what this file is or what it does. Can you explain?

(First thing I want to know is: when would early yaks explode? why? what makes them early?)

Reply:
Maybe you missed #733 which migrated things; this is a versioning mechanism and it's auto-generated to inform monorepo version bumps. https://github.com/changesets/changesets

Reply:
Confirmed with @andrewbranch that we're going to try changeset version bumps (which will cause the monorepo to not be versioned all at once). Easy to undo later if we want. Of course, the DT PR checks use `master` so I don't think I actually need to release this ASAP but it might be good to test.